src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -725,6 +725,31 @@ void createASimpleClusterWithHandle() {
             testValid("create a simple cluster with handle");
         }
 
+        @Test
+        void createEntityAndClusterWithInformativeJustification() {
+            // Two people, probably the same person
+            final String vladName = "Vladimir Putin";
+            final Resource vladimirPutin = makeEntity(model, getUri("E780885.00311"), system);
+            markName(vladimirPutin, vladName);
+
+            final Resource typeAssertion = markType(model, getAssertionUri(), vladimirPutin, SeedlingOntology.Person, system, 1.0);
+            final Resource justification = markTextJustification(model, typeAssertion, "HC00002Z0", 0, 10, system, 1d);
+            markInformativeJustification(vladimirPutin, justification);
+
+            final Resource putin = makeEntity(model, putinDocumentEntityUri, system);
+            markType(model, getAssertionUri(), putin, SeedlingOntology.Person, system, 1.0);
+            markName(putin, "Путин");
+
+            // create a cluster with prototype
+            final Resource putinCluster = makeClusterWithPrototype(model, getClusterUri(), vladimirPutin, vladName, system);
+            markInformativeJustification(putinCluster, justification);
```

Comment:
So is this saying that the same justification for the "vladimirPutin is a Person" is used as the justification that putin (with the Cyrillic name) is in the same-as cluster with vladimirPutin?  If so, I know that our examples don't have to be realistic, but should we consider creating a second justification for the cluster membership assertion?  Obviously it's "valid" either way.

Reply:
I'm content to leave it the way it is, but if you feel strongly that it should change, I can add more justifications